tests/fixtures.py

```diff
@@ -1,11 +1,11 @@
 # -*- coding: utf-8 -*-
 
-from copy import deepcopy
-
 import pytest
-
+from sys import version_info
 from pydash._compat import iteritems
 
+if version_info[0] > 2:
```

Comment:
A better way to handle this would be to use `pytest.skipif`: http://doc.pytest.org/en/latest/skipping.html#id1

```python
import pytest

from pydash._compat import PY3

@pytest.mark.skipif(not PY3, reason='requires python3')
def test_annotated_callback():
    ...
```

I'd also avoid putting actual test functions in `fixtures.py` and instead put in a `test_*.py` file. Maybe `test_annotations.py`?